kpatch-build/kpatch-build

```diff
@@ -97,24 +97,27 @@ find_dirs() {
 		# git repo
 		TOOLSDIR="$SCRIPTDIR"
 		DATADIR="$(readlink -f $SCRIPTDIR/../kmod)"
-		SYMVERSFILE="$DATADIR/core/Module.symvers"
-		return
 	elif [[ -e "$SCRIPTDIR/../libexec/kpatch/create-diff-object" ]]; then
 		# installation path
 		TOOLSDIR="$(readlink -f $SCRIPTDIR/../libexec/kpatch)"
 		DATADIR="$(readlink -f $SCRIPTDIR/../share/kpatch)"
+	fi
+}
+
+find_core_symvers() {
+	if [[ -e "$SCRIPTDIR/create-diff-object" ]]; then
+		# git repo
+		SYMVERSFILE="$DATADIR/core/Module.symvers"
+	elif [[ -e "$SCRIPTDIR/../libexec/kpatch/create-diff-object" ]]; then
+		# installation path
 		if [[ -e $SCRIPTDIR/../lib/kpatch/$ARCHVERSION/Module.symvers ]]; then
 			SYMVERSFILE="$(readlink -f $SCRIPTDIR/../lib/kpatch/$ARCHVERSION/Module.symvers)"
 		elif [[ -e /lib/modules/$ARCHVERSION/extra/kpatch/Module.symvers ]]; then
 			SYMVERSFILE="$(readlink -f /lib/modules/$ARCHVERSION/extra/kpatch/Module.symvers)"
 		else
 			warn "unable to find Module.symvers for kpatch core module"
-			return 1
```

Comment:
If we have `find_core_symvers` return 1 for an error, then I think we should move the `[[ -e "$SYMVERSFILE" ]]` check from line 454 to the very end of `find_core_symvers` and have it return 1 if the check fails. Then we could just do `find_core_symvers || die "can't find core module Module.symvers"`